features/conversation-list/use-conversation-list-conversations.tsx

```diff
@@ -49,6 +53,16 @@ export const useConversationListConversations = () => {
         topic: conversation.topic,
       })
     ),
+    // note/todo(lustig): investigate combine to remove need for filteredConvresations (which utilizes conversationsDataQueries)
+    // which is referentially unstable
+    // combine: (queries) => {
```

Comment:
read the comments - looks like combine would be the right option to prevent the issue below